pycqed/analysis_v2/dac_scan_analysis.py

```diff
@@ -629,3 +651,312 @@ def prepare_plots(self):
             'color': 'C0',
             'ax_id': 'main'
         }
+
+
+class DAC_analysis(ma.TwoD_Analysis):
```

Comment:
What is "Dac_analysis" it is not clear to me from this name. A simple docstring explaining what the experiment is it is supposed to analyze and what it will do with this is very helpful.